mm2src/common/time_cache.rs

```diff
@@ -271,6 +271,9 @@ where
     }
 
     pub fn contains(&mut self, key: &Key) -> bool { self.0.contains_key(key) }
+
+    // Removes a certain key even if it didn't expire plus removing other expired keys
+    pub fn remove(&mut self, key: Key) { self.0.remove(key); }
```

Comment:
We can inline this safely since inner remove function isn't inlined.(std remove function is already inlined and average instruction lines around 100 lines. So it wouldn't be great if we use std remove in this function and inline it).